RecoTauTag/RecoTau/plugins/TauDiscriminationAgainstElectronMVA6.cc

```diff
@@ -292,7 +358,26 @@ void TauDiscriminationAgainstElectronMVA6<TauType, TauDiscriminator, ElectronTyp
   desc.add<double>("minMVANoEleMatchWgWOgsfEC", 0.0);
   desc.add<double>("minMVAWOgWgsfEC", 0.0);
   desc.add<double>("minMVAWgWgsfEC", 0.0);
+
+  desc.add<bool>("isPhase2", false);
```

Comment:
perhaps use a switch https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#A_Switch
so that phase-2 configs are not appearing in earlier eras